README.md

```diff
@@ -6,11 +6,47 @@
 git clone https://github.com/Internet-of-People/core-control -b master
 ```
 
-### Usage
+## Important Change in 2.6.10
+
+In 2.6.10 we made three breaking changes.
+
+1. We only have a master branch now
+2. You need to prepend the `ccontrol` with a `NETWORK` environment set. See in the next section.
+3. From now on, there will be no option to start a forger node separately. You will only be able to start/restart with `relay` or `all` arg2 parameter. If you need a forger to be running, you have to use `all` which starts both the relay and the forger.
+
+You can still stop your currently running forger, but after you will only be able to start forger node together with the relay using the `all` param.
+
+#### How to Upgrade?
+
+If you ran only a relay node, you have nothing to do here. Only, if you ran a forger as well.
+
+1. Move to `master` branch.
+   ```bash
+   git fetch
+   git checkout master
+   ```
+1. Upgrade the core-control
+   ```bash
+   NETWORK=[mainnet|devnet] ccontrol update self
+   ```
+1. Upgrade Hydra Core
+   ```bash
+   NETWORK=[mainnet|devnet] ccontrol update core
+   ``` 
+1. Stop all
+   ```bash
+   NETWORK=[mainnet|devnet] ccontrol stop all
+   ```
```

Comment:
At first glance I think this is not enough, `git checkout master` will only change to the **local master without remote changes merged in**. We'd need something like `git pull` or `git merge` somewhere around here.
@mudlee Am I mistaken?

Reply:
After our discussion I agree that for people who do not have `master` yet will check out the latest version, otherwise update will do it for them.
Ok, approved.